### PR TITLE
Adding KB Article for CloudFlare SSL Termination

### PR DESCRIPTION
**Pull request by eherman-ctl**
This addition is to show how a client can use CloudFlare for SSL Termination for custom hostnames.

### CODE REVIEW

**Comment by eherman-ctl** [comment]
This addition is to show how a client can use CloudFlare for SSL Termination for custom hostnames.

File: WordPress/wordpress-cloudflare-SSL-configuration.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...

During the Limited Beta there is no production Service Level Agreement.

## Overview:
+
+There are multiple methods that could be used to terminate SSL for an existing CenturyLink Cloud WordPress site. This method covers using CloudFlare CDN's SSL Termination.
+
+### This article assumes the following:
+
+* Working knowledge of basic WordPress functionality
```
Comment (nathanyoung) [inline WordPress/wordpress-cloudflare-SSL-configuration.md]: should be `### Overview`

Comment (eherman-ctl) [inline WordPress/wordpress-cloudflare-SSL-configuration.md]: I have updated the line in question.  Thank you for your help.

Is there anything I need to do to pull this forward?

File: WordPress/wordpress-cloudflare-SSL-configuration.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## CloudFlare Settings Supported

Following the [Configure SSL on Cloud Flare] (https://support.cloudflare.com/hc/en-us/articles/204468848-How-do-I-access-or-change-any-of-my-SSL-settings-) Knowledge Base Article, these are the settings CenturyLink WordPress suports.
+
+* SSL (with SPDY)						= Full, FLexible, or Off (Strict is not currently supported)
+* HTTP Strict Transport Security (HSTS)
+ * Max Age Header (max-age)			= Any Setting
+ * Apply HSTS policy to subdomains	= On or Off
+ * Preload								= On or Off
+ * No-Sniff Header						= On or Off
```
Comment (nathanyoung) [inline WordPress/wordpress-cloudflare-SSL-configuration.md]: This one should be `###` too.

Comment (eherman-ctl) [inline WordPress/wordpress-cloudflare-SSL-configuration.md]: I have updated the line in question. Thank you for your help.

**Pull request merged**